packages/datagateway-download/src/downloadStatus/__snapshots__/adminDownloadStatusTable.component.test.tsx.snap

```diff
@@ -1092,13 +1092,13 @@ exports[`Admin Download Status Table should render correctly 1`] = `
                           <div
```

Comment:
Would be nice to remove this snapshot altogether and use normal queries instead to test the table component. I wrote some util functions in #1447 (`src/setupTests.ts`) that makes querying for table elements easier. I use them extensively in testing data tables in `dg-dataview`!

Reply:
[These are the util functions](https://github.com/ral-facilities/datagateway/blob/enhancement/rtl-migration/packages/datagateway-dataview/src/setupTests.ts#L77-L137).

Reply:
I was only here to stop the download unit tests from failing intermittently and got distracted by unit test warnings/console logging! 😅 I don't really fancy fundamentally changing any tests in this PR as that's not what it's for but I can be convinced

Reply:
No worries, I can do it in #1447 instead 👍